Update live (#529)
MicrosoftDocs/iis-docs: iis/install/web-platform-installer/web-platform-installer-direct-downloads.md

**WebPI Direct Download Links**  
  

>  In some cases, it may be helpful to be able to download the Web Platform Installer directly vs using the launcher to get started. Below are the links for the current version (WebPI 5.1).  

Reviewer: ```suggestion
 In some cases, it may be helpful to be able to download the Web Platform Installer directly rather than using the launcher to get started. The following links are for the current version (WebPI 5.1).  
```

 
**WebPI 5.1**

>  x86: [WebPI 5.1 x86 MSI] (https://go.microsoft.com/fwlink/?LinkId=287165)

Reviewer: ```suggestion
 x86: [WebPI 5.1 x86 MSI](https://go.microsoft.com/fwlink/?LinkId=287165)
```

>  x64: [WebPI 5.1 x64 MSI] (https://go.microsoft.com/fwlink/?LinkId=287166)

Reviewer: ```suggestion
 x64: [WebPI 5.1 x64 MSI](https://go.microsoft.com/fwlink/?LinkId=287166)
```
